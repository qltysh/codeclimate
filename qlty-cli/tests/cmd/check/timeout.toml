bin.name = "qlty"
args = ["check", "--all", "--no-cache"]
status.code = 1
env.remove = ["RUST_BACKTRACE"]