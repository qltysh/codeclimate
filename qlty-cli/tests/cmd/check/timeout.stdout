
 ISSUES 

:0:0
    0:0  high    timeout failed  timeout:fail

